<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GiftWidget</class>
 <widget class="QWidget" name="GiftWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="rbGiftfish">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>鱼蛋</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbfish">
         <property name="text">
          <string>100积分/10元</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="rbGiftfly">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>飞机</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbfly">
         <property name="text">
          <string>1019积分/200元</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="rbGiftrocket">
         <property name="text">
          <string>火箭</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbrocket">
         <property name="text">
          <string>16666积分/999元</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>当前金额：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdmonney">
       <property name="value" stdset="0">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>当前积分：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdintegral"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>请输入要充值的金额</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnIntegral">
       <property name="text">
        <string>积分送出</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnMonney">
       <property name="text">
        <string>余额送出</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRecharge">
       <property name="text">
        <string>充值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnGoaway">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
